app/src/main/java/de/danoeh/antennapod/adapter/NavListAdapter.java

```diff
@@ -270,7 +274,12 @@ private void bindNavView(String title, int position, NavHolder holder) {
                         new AlertDialog.Builder(context)
                             .setTitle(R.string.episode_cache_full_title)
                             .setMessage(R.string.episode_cache_full_message)
-                            .setPositiveButton(android.R.string.ok, (dialog, which) -> { })
+                            .setPositiveButton(R.string.open_autodownload_settings, (dialog, which) -> {
+                                Intent intent = new Intent(context, PreferenceActivity.class);
+                                intent.putExtra(OPEN_AUTO_DOWNLOAD_SETTINGS, true);
+                                context.startActivity(intent);
+                            })
+                            .setNegativeButton(R.string.cancel_label, null)
```

Comment:
I would show an OK button (positive) and the settings button (neutral) instead. The main action users should do on that dialog is get informed why it is not downloading. They should not be nudged to increase the cache size more and more, so the settings button should not be the main action.